lib/map.js

```diff
@@ -105,6 +105,13 @@ define(['map/clientlayer', 'map/labellayer', 'map/button', 'leaflet'],
         }
       });
 
+      map.on('load', function () {
+        document.querySelectorAll('.leaflet-control-layers-selector').forEach(function (input) {
+          input.setAttribute('role', 'radiogroup');
+          input.setAttribute('aria-label', input.nextSibling.innerHTML.trim());
```

Comment:
Hat kein Effekt oder ist nicht zu sehen

Reply:
![image](https://user-images.githubusercontent.com/1295633/32067698-6a6a4be4-ba84-11e7-8dff-c20c961e3815.png)

Reply:
joa, denk schon das es klappt - doch was für eine auswirkung hat es auf den browser.
Oder ist es wirklich nur für spezielle accessbility addon

Reply:
screen reader usw. and usability addons